compiler/rustc_sanitizers/src/cfi/typeid/itanium_cxx_abi/encode.rs

```diff
@@ -746,7 +746,7 @@ fn to_disambiguator(num: u64) -> String {
 /// <https://itanium-cxx-abi.github.io/cxx-abi/abi.html#mangle.seq-id>).
 fn to_seq_id(num: usize) -> String {
     if let Some(num) = num.checked_sub(1) {
-        base_n::encode(num as u128, 36).to_uppercase()
+        base_n::encode(num as u128, base_n::CASE_INSENSITIVE).to_uppercase()
```

Comment:
the link right above says
> A <[seq-id](https://itanium-cxx-abi.github.io/cxx-abi/abi.html#mangle.seq-id)> is a sequence number in base 36,
so i'm not convinced that this is an improvement either. but whatever, it's fine